Deployment/Get-FunctionUri.ps1

```diff
@@ -7,28 +7,22 @@
     NOTE: Invoke-RestMethod may fail if wrong SecurityProtocol is used.
     Tls12 should work
 
-    .PARAMETER ResourceGroup
-    Name of the resource group that has the web app deployed
-
-    .PARAMETER WebAppName
-    Name of the target web app
+    .PARAMETER WebApp
+    PSSite web app
 
     .EXAMPLE
     .\Publish.ps1 -ResourceGroup "github-test" -WebAppName "test-app"
 #>
 param(
-    [Parameter(Mandatory = $true)][string]$ResourceGroup,
-    [Parameter()][string]$WebAppName = $ResourceGroup,
+    [Parameter(Mandatory = $true)][Microsoft.Azure.Commands.WebApps.Models.PSSite]$WebApp,
```

Comment:
There's shortcut [Parameter(Mandatory)] eg. available for flags without need of = $true/$false